src/about-menu/open-about-menu.ts

```diff
@@ -35,17 +37,31 @@ export function openAboutWindow() {
 	// Read package.json and pass version to about window
 	const packageJson = app.getVersion();
 
+	function escapeSingleQuotes( str: string ) {
+		return str.replace( /'/g, "\\'" );
+	}
```

Comment:
Nice! I tested it and it renders the single quote and it doesn't break the About page.
I tried replacing to `&quot;`, but it didn't render the correct entity.

<img width="396" alt="Screenshot 2024-06-11 at 13 48 14" src="https://github.com/Automattic/studio/assets/779993/a4600e1a-70fc-4c98-975d-984fb2a8e4ed">

I've also seen that `@wordpress/escape-html` `escapeAttribute` Only escapes, the `>`, `"` and `&` symbols.